<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondScreen</class>
 <widget class="QDialog" name="SecondScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>218</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Receiver</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="receiverEmailLabel">
        <property name="text">
         <string>Receiver Email:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="receiverEmailEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="receiverIdLabel">
        <property name="text">
         <string>Receiver Id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="receiverIdEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="amountSendLabel">
        <property name="text">
         <string>Amount to Send:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="amountSendEdit"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="sendScreenButton">
        <property name="text">
         <string>Send </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
